### ISSUE TITLE
Update in Extension

### ISSUE THREAD

**Issue by shivamkumarc** [elliotwaite/thumbnail-rating-bar-for-youtube#64]
Can you make show number of likes visible near the like percentage of the video or replace the like percentage with the number of likes on the video

**Comment by elliotwaite** [comment]
Thanks for the suggestion. I'm currently not sure if I want to add this option to the extension since there is already the option to hover over the rating bar to see a tooltip of the like/dislike counts, and I feel like also showing the counts next to the rating percentage might start to feel like too much added text. But if enough other people also want this feature added, I'll consider adding it. So I'll leave this issue open for now to gauge interest.

Also, if you wanted to customize the extension to your liking, you can always download or clone the repo and add this specific feature, and then load the extension locally in your browser (let me know if you need help with this). You would just need to update the [`addRatingPercentage`]() function to also add the like/dislike counts after the percentage: https://github.com/elliotwaite/thumbnail-rating-bar-for-youtube/blob/master/extension/content-script.js#L324

**Comment by shivamkumarc** [comment]
Thanks for the reply sir, till the update I want to do this locally, I want
to remove like percentage and add a number of likes instead of how to do
that sir

**Comment by shivamkumarc** [comment]
Sir I'm not able to download it can you make a video to explain whole
process or just explain in chat how to download that script

**Comment by elliotwaite** [comment]
To download the repo, go [here](https://github.com/elliotwaite/thumbnail-rating-bar-for-youtube), then click the green "Code" button near the top right, then click Download ZIP.

I don't have time to make a video about this, but you should be able to find educational resources online about any of the steps I described above to better understand them.

**Comment by shivamkumarc** [comment]
Sir it still does not show number of likes near title of the video

**Comment by shivamkumarc** [comment]
Sir, it still does not show the number of likes near the title of the video
what I've done sir is here: https://youtu.be/gyP_GViVNak

**Comment by shivamkumarc** [comment]
Sir please can you make the video explaining the whole process and send it
to me on WhatsApp number 9110169662

**Comment by elliotwaite** [comment]
You still have to enable the "color-coded text percentage" option.

**Comment by shivamkumarc** [comment]
I`ve enabled sir like percentage but no of likes not showing

**Comment by elliotwaite** [comment]
Did you also update the `getRatingPercentageHtml` function code?

**Comment by shivamkumarc** [comment]
Sir can you make a separate extension whose name be like youtube likes
shower and the extension job will be only to show the number of likes near
the title of the video

**Comment by shivamkumarc** [comment]
how to do that sir,i only did what i showed in youtube video

**Comment by elliotwaite** [comment]
In the extension directory, there is a file named `content-script.js`. Open it in a text editor and find the code for the `getRatingPercentageHtml` function, and replace it as I described above, then save that file and click the refresh button for the extension (the circular arrow one).

**Comment by shivamkumarc** [comment]
I`ve made the change but now how to download it

**Comment by shivamkumarc** [comment]
Sir  content-script.js
<https://github.com/elliotwaite/thumbnail-rating-bar-for-youtube/blob/master/extension/content-script.js#L190>
file
is not opening what to do:

**Comment by elliotwaite** [comment]
You need a text editor like VSCode or Sublime Text. You can look up YouTube videos about how to edit JavaScript files.

**Comment by shivamkumarc** [comment]
Finally done sir thank you thank you thank you so much can you also add the
color to the like and dislike number, I will change the code again to
content-script.js
<https://github.com/elliotwaite/thumbnail-rating-bar-for-youtube/blob/master/extension/content-script.js#L190>
when
you will provide new code

**Comment by elliotwaite** [comment]
Try this:
```
function getRatingPercentageHtml(videoData) {
  let r = (1 - videoData.rating) * 1275
  let g = videoData.rating * 637.5 - 255
  if (!isDarkTheme) {
    g = Math.min(g, 255) * 0.85
  }
  let rgb = 'rgb(' + r + ',' + g + ',0)'
  let text = videoData.likes.toLocaleString() + ' likes, ' + videoData.dislikes.toLocaleString() + ' dislikes'

  return '<span class="style-scope ytd-video-meta-block ytrb-percentage" style="color:' +
      rgb + ' !important">' + text + '</span>'
}
```

**Comment by shivamkumarc** [comment]
Thank you  Thank you  Thank you  Thank you  Thank you  Thank you  Thank
you  Thank you  Thank you  Thank you  Thank you  Thank you  Thank you
Thank you  Thank you  Thank you so much sir

**Comment by elliotwaite** [comment]
Glad to hear you got it working. Should I close this issue now?

**Comment by elliotwaite** [comment]
Sounds like a good strategy.

**Comment by shivamkumarc** [comment]
Thank you!

**Comment by elliotwaite** [comment]
You're welcome.

**Comment by shivamkumarc** [comment]
Sir likes and dislike data is loading for every video except when we search
any topic, in people  also watched section only the last video like dislike
data is loading and in, for you section also only last video like dislike
data is loading,else it is doing perfect job in loading like dislike data

**Comment by elliotwaite** [comment]
Okay, I'm not sure how to fix that.

**Comment by shivamkumarc** [comment]
Ok, no problem it's only two to three videos are such

**Comment by elliotwaite** [comment]
I've updated the title of this issue to be "Add the number of likes/dislikes below the title of the video now", which corresponds with your latest comment (https://github.com/elliotwaite/thumbnail-rating-bar-for-youtube/issues/67), and I think that is a more accurate description of your feature request. But feel free to update the title again if you think it should be something else.

**Comment by shivamkumarc** [comment]
Sir but it still shows like percentage and not the number of likes and
dislikes

**Comment by elliotwaite** [comment]
That's correct. Please refer to: https://github.com/elliotwaite/thumbnail-rating-bar-for-youtube/issues/64#issuecomment-1013956274

**Comment by shivamkumarc** [comment]
Sir I've already done it locally,can you do this to main extension itself

**Comment by shivamkumarc** [comment]
I`ve already done it locally sir

**Comment by elliotwaite** [comment]
I do not want to add this feature at the moment. If enough other people also want this feature added, I will consider it.

**Comment by shivamkumarc** [comment]
ok sir thanks for the reply

**Comment by elliotwaite** [comment]
No problem. I hope that is understandable.

**Comment by shivamkumarc** [comment]
Yes sir

**Comment by shivamkumarc** [comment]
Sir can you make some changes to the local extension as the thumbnail
rating bar loads for every video but like and dislike data below the title
of the video for some videos it doesn`t load

**Comment by elliotwaite** [comment]
Which videos does it not load for? Is it a certain type of thumbnail, or just every once in a while it doesn't work?

**Comment by shivamkumarc** [comment]
For some videos on the home page, it doesn`t load but thumbnail rating bar
loads for every video

**Comment by shivamkumarc** [comment]
for example for right hand side video didn`t load

**Comment by elliotwaite** [comment]
Can you post a screenshot?

**Comment by shivamkumarc** [comment]
And for also these two videos,it didn`t load

**Comment by elliotwaite** [comment]
I think something went wrong when teid to post those screenshots. I only see text.

**Comment by shivamkumarc** [comment]
Sir, I`ve downloaded the latest code of extension and made the change you
told to me but still, it`s not loading for some of the videos, what I
shared screenshot is on the videos section of the youtube channel,
otherwise for the home page and when we search it loads for maximum of the
videos but don`t load for people also watched,for you,previously
watched,top news section
